packages/cli/src/lib/getNodeOptions.js

```diff
@@ -0,0 +1,22 @@
+// Exported for testing.
+export const enableSourceMapsOption = '--enable-source-maps'
+
+/**
+ * Gets the NODE_OPTIONS environment variable from `process.env`, appending `--enable-source-maps` if it's not already there.
+ * See https://nodejs.org/api/cli.html#node_optionsoptions.
+ *
+ * @returns {string}
+ */
+export function getNodeOptions() {
```

Comment:
Nitpick Dom, should you pass in enable source maps and make this function mergeNodeOptions?